### PR TITLE
WIP add iso-ara-Arab-Latn-223-1993

### PR DESCRIPTION
**Pull request by AhMohsen46**

### CODE REVIEW

File: maps/iso-ara-Arab-Latn-233-1993.yaml [hunk 1]
```diff
@@ -0,0 +1,180 @@
+---
+authority_id: iso
+id: 233-2:1993
+language: iso-639-2:ara
+source_script: Arab
+destination_script: Latn
+name: ISO 233:1993 Documentation — Transliteration of Arabic characters into Latin characters
+url: https://cdn.standards.iteh.ai/samples/4118/2f03c828842c4055a5619c1bded39381/ISO-233-2-1993.pdf
```
Comment (ronaldtse) [inline maps/iso-ara-Arab-Latn-233-1993.yaml]: Please change this to the official ISO url.

**Review by ronaldtse — CHANGES_REQUESTED** [review]

**Comment by ronaldtse** [comment]
@AhMohsen46 are we ready to merge this? Thanks.

**Comment by AhMohsen46** [comment]
Yes I believe this is complete
Will check the docs the earliest next morning (in 6 hours at most)

**Comment by ronaldtse** [comment]
I've migrated this to the new structure here: interscript/maps#3 . Closing this.

**Pull request closed without merging**